telemetry/definitions/commonDefinitions.json

```diff
@@ -855,6 +866,11 @@
             "description": "Open a view of a S3 bucket",
             "metadata": [{ "type": "result" }]
         },
+        {
+            "name": "s3_editObject",
+            "description": "Edit or view one or more S3 objects in the IDE editor",
```

Comment:
What do you think about different metrics for the operations of "edit" and "view" against an s3 object? This may be confusing to interpret in the future.

Reply:
Sounds good

Reply:
@awschristou we intentionally do not have different metrics. The point is that the action is the same, but it has different flavors.

It is noisy to have "view" and "edit" as separate metrics in dashboards. For cases where we want the distinction, the metadata gives that ability.